package.json

```diff
@@ -31,18 +31,18 @@
   "devDependencies": {
```

Comment:
should we update the `version` field if we plan to ship this change?

Reply:
we have other changes as well. so will do the version release in a separate PR.